docs/install/_rpi.rst

```diff
@@ -1,52 +1,83 @@
+
+Kolibri is intended for **Raspberry Pi Model 3** and upwards.
+
+.. note:: The standard Raspbian OS has a graphical desktop. You can also install Raspbian Lite which uses fewer resources, but only has a command line interface. The instructions in this documentation work seamlessly on both.
 
-Kolibri works well on Debian-based distributions for `Raspberry Pi <https://www.raspberrypi.org/>`_, such as Raspbian, and has been tested on RPi 3 models.
 
 Install
 -------
 
-1. Running ``add-apt-repository`` as shown in the PPA instructions does not work. Instead, run::
+#. We need to upgrade the ``python3-cffi`` library, which is outdated on Raspbian. Upgrade it like this:
+
+   .. code-block:: bash
+
+      sudo apt install libffi-dev
+      sudo pip3 install pip --upgrade
+      sudo pip3 install cffi --upgrade
+      sudo systemctl start kolibri
+
+  .. warning:: If you are installing Kolibri on the older **Raspbian Jessie** which is from before August 2017, you need to complete a few extra steps:
+  
+   .. code-block:: bash
+    
+        sudo apt install python3-pip python3-pkg-resources
+        sudo pip3 install setuptools --upgrade
+
+#. Add our Ubuntu PPA with these special instructions:
+
+   .. code-block:: bash
 
       sudo apt install dirmngr
       sudo su -c 'echo "deb http://ppa.launchpad.net/learningequality/kolibri/ubuntu xenial main" > /etc/apt/sources.list.d/learningequality-ubuntu-kolibri-xenial.list'
       sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys DC5BAA93F9E4AE4F0411F97C74F88ADB3194DD81
       sudo apt update
       sudo apt install kolibri
 
-2. Kolibri does not start after installation. This is because ``python3-cffi`` is outdated on Raspbian. Upgrade it like this::
+#. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
 
-      sudo apt install libffi-dev
-      sudo pip3 install pip --upgrade
-      sudo pip3 install cffi --upgrade
-      sudo systemctl start kolibri
 
-3. When the command finishes, open the default browser at http://127.0.0.1:8080 and proceed with the :ref:`setup_initial` of your facility. 
+.. note:: The following issues are quite common on a Raspberry Pi:
+
+  * **System time** isn't set properly or resets during power-off. This causes errors while downloading software. For instance, SSL certificates for online sources will fail to validate. Ensure that you have the right timezone in ``/etc/timezone`` and that the clock is set properly by running ``sudo ntpd -gq``.
 
+  * **Storage space** is often a scarce. If you have a USB source for additional storage, you can use a ``kolibri`` management command or create your own symbolic links to have the data folder located elsewhere.
```

Comment:
is often scarce

:)

Reply:
Argh, this is sloppy, sorry - was formatting SD cards and jumping back and forth on the KVM switch :) Fixed!

Reply:
better than "is often a farce"

Reply:
:smile: